utils/codegen/codegen-rust/rgen/src/conversion/utils.rs

```diff
@@ -158,7 +158,7 @@ impl Conversion {
             ASN1Type::Integer(i) => (i.constraints.clone(), "INTEGER".into()),
             ASN1Type::Real(_) => (vec![], "float64".into()),
             ASN1Type::ObjectIdentifier(_o) => todo!(),
-            ASN1Type::BitString(_b) => todo!(),
+            ASN1Type::BitString(b) => (b.constraints.clone(), "uint8[]".into()),
```

Comment:
Check if correct. Why did this never get raised for the other message types?

Same for change in `utils/codegen/codegen-rust/rgen/src/msgs/utils.rs`.